### DISCUSSION TITLE
How to populate IMEI value Type 20

### DISCUSSION THREAD

**Discussion by Chiragdeshlehra27** [zeek/zeek#3901] [Help] [answered]
I have added a coloum in radius log with a name IMEI and want to get the populated values but the catch is the value comes under Type 20 vendor specific, So for me pulling Type 20 is easy but with this nested condition Type 26 then inside it Type 20

**Comment by bbannier** [comment, marked as answer]
Hi,

We do not provide user support in GitHub discussions. Questions like this are best answered using Slack, or our forum. Both of these are followed by Zeek developers and professional users. You can join here:

Forum: https://community.zeek.org/

Slack: https://zeek.org/community/
